docs/tui-demo/async.md

```
@@ -0,0 +1,579 @@
+    # or, for everything:
+    pip install sqliter-py[full]
+    ```
+
+---
+
+## Async Connection
+
+Create an async database connection. The constructor parameters are identical to
+`SqliterDB` — only the method calls need `await`.
+
+```python
```

markdownlint-cli2 (0.22.1)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)